financial-connections/src/main/java/com/stripe/android/financialconnections/features/MarkdownParser.kt

```diff
@@ -1,74 +1,21 @@
 package com.stripe.android.financialconnections.features
 
-import com.stripe.android.financialconnections.model.Bullet
-import com.stripe.android.financialconnections.model.ConsentPane
-import com.stripe.android.financialconnections.model.ConsentPaneBody
-import com.stripe.android.financialconnections.model.DataAccessNotice
-import com.stripe.android.financialconnections.model.DataAccessNoticeBody
-import com.stripe.android.financialconnections.model.LegalDetailsBody
-import com.stripe.android.financialconnections.model.LegalDetailsNotice
-
 internal object MarkdownParser {
-    private val markDownToHtmlRegex = listOf(
+
+    private val markDownToHtmlRegex = listOf<Pair<Regex, (MatchResult) -> CharSequence>>(
         // bold, italics rules
-        "\\*\\*\\s?([^\\n]+)\\*\\*".toRegex() to "<b>$1</b>",
-        "__([^_]+)__".toRegex() to "<b>$1</b>",
+        "\\*\\*(.*?)\\*\\*".toRegex() to { "<b>${it.groupValues[1]}</b>" },
+        "__([^_]+)__".toRegex() to { "<b>${it.groupValues[1]}</b>" },
```

Comment:
There was an issue with multiple bolded markdown sections in the same string. Added tests to cover it.

Reply:
can we define them as constants with a propername?